models/questions.js

```diff
@@ -2,46 +2,49 @@ module.exports = {
   name: "questions",
   schema: {
     externalId: {
-      type: String,
+      type: "String",
       index: true,
       unique: true
     },
-    question: Array,
-    tip: String,
-    hint: String,
-    responseType: String,
-    value: String,
-    isCompleted: { type : Boolean, default: false },
-    showRemarks: { type : Boolean, default: false },
-    remarks: String,
-    visibleIf: Object,
-    createdBy: String,
-    updatedBy: String,
-    options: Array,
-    sliderOptions: Array,
-    children: Array,
-    questionGroup: Array,
-    questionType: String,
-    modeOfCollection: String,
-    usedForScoring: String,
-    file: Object,
-    fileName: Array,
-    validation: Object,
-    accessibility: String,
-    instanceIdentifier: String,
-    noOfInstances: Number,
-    notApplicable: String,
-    canBeNotApplicable: String,
-    instanceQuestionsString: String,
-    instanceQuestions: Array,
-    isAGeneralQuestion: { type : Boolean, default: false },
-    dateFormat: String,
-    autoCapture: { type : Boolean, default: false },
-    rubricLevel: String,
-    sectionHeader: String,
-    allowAudioRecording: Boolean,
-    page: String,
-    questionNumber: String,
-    weightage: { type : Number, default: 1 }
+    question: "Array",
```

Comment:
@aman-tunerlabs - What is the reason for putting everything in between quotes and converting it all to a string ?

Reply:
So that i can check the type as Boolean string otherwise it will show a functions. And for the rest of the type like String,and array to make it consistent.

Reply:
@aman-tunerlabs - Can we derive from that function whether or not it is boolean? Else we can add one more extra parameter in the model itself which we can use ?